srcpkgs/oomox/template

```diff
@@ -22,12 +22,12 @@ distfiles="
  https://github.com/nana-4/materia-theme/archive/$_materia_theme_ver.tar.gz>materia-theme-$_materia_theme_ver.tar.gz
```

Comment:
Add the above removed `v` here `...archive/v$_materia_theme_ver.tar.gz>...`

Reply:
so like `https://github.com/nana-4/materia-theme/archive/v$_materia_theme_ver.tar.gz>materia-theme-v$_materia_theme_ver.tar.gz`? or should it be `https://github.com/nana-4/materia-theme/archive/v$_materia_theme_ver.tar.gz>materia-theme-$_materia_theme_ver.tar.gz`

Reply:
I think it should be
`https://github.com/nana-4/materia-theme/archive/v$_materia_theme_ver.tar.gz>materia-theme-$_materia_theme_ver.tar.gz`
so that the new archive name corresponds to `$pkgname-$version.tar.gz`.